pkg/clicmd/prototype_use.go

```diff
@@ -75,6 +75,16 @@ expand prototypes into Jsonnet files.
 command line flags, such as ` + " `--image` " + `in the example above. Note that
 different prototypes support their own unique flags.
 
+You can also use a values file to supply values to the prototype. The format for the
+file is a key and value per line.
+
+		key1=value1
```

Comment:
We should be explicit about what characters are and aren't allowed in keys and values. For example, do we need to support `=` characters in values?

Reply:
ksonnet accept any unicode character if the Param is designated as a string.